pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/BuildFinderUtils.java

```diff
@@ -172,10 +171,10 @@ public static BuildConfig getKojiBuildFinderConfig() {
      */
     public static List<KojiBuild> findBuilds(File file, boolean includeNotFound) {
         BuildConfig config = getKojiBuildFinderConfig();
-        List<File> inputs = Collections.singletonList(file);
-        ExecutorService pool = Executors.newFixedThreadPool(2);
+        List<String> inputs = Collections.singletonList(file.getPath());
+        ExecutorService pool = Executors.newFixedThreadPool(config.getChecksumTypes().size());
```

Comment:
Pool size here should just be 2 (1 for analyzer, 1 for finder). There was an error in previous versions of the code, so revert this change.

Reply:
Well spotted, sorry that got in with a merge conflict.